packages/payment-detection/src/thegraph/client.ts

```diff
@@ -96,14 +97,17 @@ export const defaultGetTheGraphClient = (
   return network === 'private'
     ? undefined
     : NearChains.isChainSupported(network)
-    ? getTheGraphNearClient(`${HOSTED_THE_GRAPH_URL}${network.replace('aurora', 'near')}`, options)
+    ? getTheGraphNearClient(
+        `${THE_GRAPH_STUDIO_URL.replace('$NETWORK', network.replace('aurora', 'near'))}`,
+        options,
+      )
     : network === 'mantle'
     ? getTheGraphEvmClient(THE_GRAPH_URL_MANTLE, options)
     : network === 'mantle-testnet'
     ? getTheGraphEvmClient(THE_GRAPH_URL_MANTLE_TESTNET, options)
     : network === 'core'
     ? getTheGraphEvmClient(THE_GRAPH_URL_CORE, options)
-    : network === 'zksyncera' || network === 'base'
-    ? getTheGraphEvmClient(`${STUDIO_THE_GRAPH_URL}${network}/version/latest`, options)
-    : getTheGraphEvmClient(`${HOSTED_THE_GRAPH_URL}${network}`, options);
+    : ['mainnet', 'xdai'].includes(network)
+    ? getTheGraphEvmClient(`${THE_GRAPH_STUDIO_URL_RF.replace('$NETWORK', network)}`, options)
```

Comment:
Please consider adding a comment explaining why mainnet and xdai are on the RF Subgraph Studio account instead of the RN Subgraph Studio account.